dashboard/test/controllers/curriculum_proxy_controller_test.rb

```diff
@@ -13,4 +13,14 @@ class DocsProxyControllerTest < ActionController::TestCase
     assert_response :success
     assert_equal response.body, 'docs.code.org content'
   end
+
+  test "should redirect from studio.code.org/curriculum path to curriculum.code.org path" do
+    stub_request(:get, "https://curriculum.code.org/csd/maker_leds/index.html").
+      to_return(body: 'curriculum.code.org content', headers: {})
+
+    request.host = "studio.code.org"
+    get :get, params: {curriculum_route: 'csd/maker_leds/index.html'}
```

Comment:
As stated above, it seems this param value isn't actually used. It seems misleading. What if I pass a different `curriculum_route` from the requested path? I'd prefer removing this param.

We should add a test for the 500 error too (neither a docs nor curriculum route), since that's an option. If we remove those params as I'd prefer than it might no longer be a valid case.

Reply:
Locally, I've changed my code to do `if request.path.start_with?('/docs')` instead of looking at `params[:docs_route]`, and seem to have that working fine.

However, I'm struggling with how to write a test for this.

If I have:
```
get :get, params: {path: 'csd/maker_leds/index.html'}
```
request.path ends up starting with `/docs`, but I'm (a) not sure why and (b) not sure how to then test the /curriculum path as well. Any suggestions?